<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Imager</class>
 <widget class="QWidget" name="Imager">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imager</string>
  </property>
  <widget class="QPushButton" name="rec_button">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color : white;border-radius: 5px;border-style: outset;border-width: .5px;border-color: gray;}QPushButton::pressed{border-style: inset;border-color: gray;background-color : red;}</string>
   </property>
   <property name="text">
    <string>Rec</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stop_button">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{background-color : white;border-radius: 5px;border-style: outset;border-width: .5px;border-color: gray;}QPushButton::pressed{border-style: inset;border-color: gray;}</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fps_input">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>40</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>60</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QComboBox" name="colormaps">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>101</width>
     <height>32</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>gray</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>jet</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fps</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="frames">
   <property name="geometry">
    <rect>
     <x>313</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <italic>true</italic>
    </font>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="smallDecimalPoint">
    <bool>false</bool>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="value">
    <number>600</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>70</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>colormap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>size</string>
   </property>
  </widget>
  <widget class="QSlider" name="exposure_input">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>150</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>130</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>exposure</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
